plugins/thunderbolt/fu-thunderbolt-firmware.c

```diff
@@ -382,6 +384,8 @@ fu_thunderbolt_firmware_parse (FuFirmware *firmware,
 		{ 0x15EA, 3, _FAMILY_TR, 2 }, /* TR 4C */
 		{ 0x15EF, 3, _FAMILY_TR, 2 }, /* TR 4C device */
 		{ 0x15EE, 3, _FAMILY_BB, 0 }, /* BB device */
+		{ 0x1136, 4, _FAMILY_MR, 2 }, /* MR DP */
+		{ 0x1137, 4, _FAMILY_MR, 2 }, /* MR 4C device */
```

Comment:
no idea on this bit unfortunately, but the rest looks sane.

Reply:
Yeah I think @westeri will need to confirm these and the descriptions for them.

Reply:
Hi, actually in Windows they use UEFI Capsule update for MR so this is what we should do in Linux too . That's the reason the kernel side does not allow NVM update through the driver. I should have communicated this with you guys earlier, sorry about that.

Reply:
@westeri oh right.  I knew that in the back of my mind.
Also, what about the missing 0x1136 in the kernel for that card though?  Should there be something similar to https://git.kernel.org/pub/scm/linux/kernel/git/westeri/thunderbolt.git/commit/?h=next&id=db0746e3399ee87ee5f957880811da16faa89fb8 for that ID?


@hughsie up to you, do we leave the IDs so that the parser can still understand them in case a non-UEFI MR case comes up later?  It would still require kernel code changes though to allow flashing.

Reply:
> so that the parser can still understand them in case a non-UEFI MR case

I think that's sensible, yes.

Reply:
> I think that's sensible, yes.

OK let me change the PR to add a comment to the code for future selfs to know why we left it then.

Reply:
Hi, 0x1136 is the PCIe downstream port of MR. I don't think it needs to be added to the TBT driver.

Reply:
OK thanks!